iOS/classes/BleAdapter.swift

```diff
@@ -0,0 +1,156 @@
+import Foundation
+
+@objc public protocol BleAdapter {
+
+    // MARK: - Properties
+
+    var delegate: BleClientManagerDelegate? { get set }
+
+    // MARK: - Initializers
+
+    init(queue: DispatchQueue, restoreIdentifierKey: String?)
+    func invalidate()
+
+    // MARK: - Functions
+
+    func cancelTransaction(_ transactionId: String)
+
+    func setLogLevel(_ logLevel: String)
+
+    func logLevel(_ resolve: Resolve,
+                  reject: Reject)
+
+    func enable(_ transactionId: String,
+                resolve: Resolve,
+                reject: Reject)
+
+    func disable(_ transactionId: String,
+                 resolve: Resolve,
+                 reject: Reject)
+
+    func state(_ resolve: Resolve,
```

Comment:
is it clear that it is state of radio?

Reply:
It's the naming of the BleClientManager methods so I wanted to keep the convention and do not change method names (I also think it might be a little confusing)